quodlibet/ext/events/stop_on_pause.py

```diff
@@ -23,15 +23,15 @@ class StopOnPause(EventPlugin, PluginConfigMixin):
     PLUGIN_ICON = Icons.MEDIA_PLAYBACK_PAUSE
     PLUGIN_DESC = _("This plugin changes Pause to Stop/Play/Seek."
                     "\n\n"
-                    "This behavior is useful if Quod Libet fails to switch "
-                    "audio devices when a new audio device is added to the "
-                    "system (Bluetooth speaker, USB DAC, etc)."
+                    "Useful if Quod Libet can\'t switch audio devices "
+                    "when a new one (Bluetooth speaker, USB DAC, etc.) "
+                    "is added to the system."
                     "\n\n"
-                    "If the option for 'Only Seekable Sources' "
-                    "is not checked, this plugin will ensure the position in "
-                    "the current source is maintained. Otherwise, playback "
-                    "will be left stopped and must restart from the begining.")
-
+                    "Ensures position in the current source is maintained "
+                    "so long as 'Only Seekable Sources' is off. "
+                    "Otherwise, playback is left stopped and must be restarted "
+                    "from the begining.")
+    
```

Comment:
The white space here is the reason for the test failure.

Reply:
:)